nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenHTTP.java

```
@@ -373,6 +386,14 @@ public void createHttpServer(final ProcessContext context) throws Exception {
 
     @Override
     public void onTrigger(final ProcessContext context, final ProcessSessionFactory sessionFactory) {
+        try {
+            if (!initialized.get()) {
+                createHttpServerFromService(context);
+            }
+        } catch (Exception e) {
+            context.yield();
```

We should not swallow this exception without logging at least logging it.